### PR TITLE
Minor changes in pom

### PR DESCRIPTION
**Pull request by Sammers21**
* extracting fabric8-docker-plagin.version property 
* changed commentary for $SERVER property

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -24,8 +24,9 @@
     <stack.version>3.5.0-SNAPSHOT</stack.version>
     <junit.version>4.12</junit.version>
     <paho.version>1.1.0</paho.version>
+    <fabric8-docker-plagin.version>0.22.1</fabric8-docker-plagin.version>
```
Comment (ppatierno) [inline pom.xml]: Fix typo "plagin" to "plugin" ;)

Comment (Sammers21) [reply pom.xml]: Sure

File: pom.xml [hunk 2]
```diff
@@ -24,8 +24,9 @@
     <stack.version>3.5.0-SNAPSHOT</stack.version>
     <junit.version>4.12</junit.version>
     <paho.version>1.1.0</paho.version>
+    <fabric8-docker-plugin.version>0.22.1</fabric8-docker-plugin.version>
```
Comment (ppatierno) [inline pom.xml]: Out of curiosity. I'm ok with that but do you have increased the version for any specific reason ?

Comment (Sammers21) [reply pom.xml]: No specific reason, just a newest version

**Pull request merged**